src/lib/seam/thermostats/climate-setting-schedules/use-delete-climate-setting-schedule.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  useMutation,
+  type UseMutationResult,
+  useQueryClient,
+} from '@tanstack/react-query'
+import type { ClimateSettingScheduleDeleteRequest, SeamError } from 'seamapi'
+
+import { NullSeamClientError, useSeamClient } from 'lib/seam/use-seam-client.js'
+
+export type UseDeleteClimateSettingScheduleMutationParams =
+  ClimateSettingScheduleDeleteRequest
+
+export function useDeleteClimateSettingSchedule(): UseMutationResult<
+  unknown,
```

Comment:
The endpoint returns `{}`, is `unknown` the recommended return type here?

Reply:
How about via https://github.com/typescript-eslint/typescript-eslint/issues/2063#issuecomment-675156492
```suggestion
  Record<string, never>,
```

Reply:
Just kidding the sdk returns `{ok: true}`, I was looking at connect

https://github.com/seamapi/react/pull/364/commits/6f9e86c93f0bca4ea30b137122b4cff1a723dfee